qemu/tests/tpm_verify_device.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import re
+
+from avocado.utils import process
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Verify the TPM device info inside guest and host.
+    Steps:
+        1. Boot guest with a emulator TPM device or pass through device.
```

Comment:
Suggest change to:
Boot guest with a emulated TPM device or passthrough TPM device.

Reply:
Thanks, have updated.